src/test/java/com/iyzipay/functional/builder/OrderItemBuilder.java

```diff
@@ -0,0 +1,88 @@
+package com.iyzipay.functional.builder;
+
+import com.iyzipay.model.OrderItem;
+import com.iyzipay.model.OrderItemType;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class OrderItemBuilder {
```

Comment:
bu builder'ı sadece teste değil koda da ekleyebiliriz. müşterinin işini kolaylaştıracaksa tabii.

Reply:
Diğer builder'lar burda olduğu için buraya ekledik. Genel yapıyı koruyalım dedik.